explainer.md

```diff
@@ -74,14 +74,15 @@ encrypted and congestion-controlled communication.
 const host = 'example.com';
 const port = 10001;
 const transport = new QuicTransport(host, port);
+const datagramWritableStream = transport.sendDatagrams();
 
 setInterval(() => {
   // App-specific encoded game state
   const gameState = getGameState();
   const encodeGameState = encodeGameState(gameState);
-  try {
-    transport.sendDatagram(encodedGameState);
-  } catch(err) {
+  if (datagramWritableStream.ready) {
```

Comment:
ready is a promise attribute. I think there's two ways to do this.
1. await the ready promise and if it resolves before a delta 100 ms you write(). 
2. don't write unless the last promise has resolved 

2. is more simple but 1. gives more accuracy (makes sure you aren't buffering a datagram).
https://streams.spec.whatwg.org/#default-writer-ready

Reply:
Why don't we just say that if you call write when there is back pressure you get dropped (unlike a stream, where you get queued)?  Then we can just call .write() without checking .ready.